pylib/pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[project]
+name = "anki"
+# dynamic = ["version"]
+version = "0.1.2"
+requires-python = ">=3.9"
+license = "AGPL-3.0-or-later"
+dependencies = [
+  "beautifulsoup4",
+  "decorator",
+  "markdown",
+  "orjson",
+  "protobuf>=4.21",
+  "requests[socks]",
+  "typing_extensions",
+  "types-protobuf",
+  "types-requests",
+  "types-orjson",
+  # platform-specific dependencies
+  "distro; sys_platform != 'darwin' and sys_platform != 'win32'",
+  "psutil; sys_platform == 'win32'",
+]
+
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.hatch.build.targets.wheel]
+packages = ["anki"]
+
+[tool.hatch.version]
+source = "code"
+path = "../python/version.py"
+
+[tool.hatch.build.hooks.custom]
+path = "hatch_build.py"
```

Comment:
@dae  Should `pylib/test/` and anything else be excluded? https://hatch.pypa.io/latest/build/#configuration